### PR TITLE
Docs: adding information on Receipt Context  and Glossary

### PR DESCRIPTION
**Pull request by alynn26**
Adding information on Receipt Template Context.

### CODE REVIEW

File: docs/admin/receipt_template_editor.adoc [hunk 1]
```diff
@@ -228,15 +232,14 @@ once you import the receipts on the new machine.
 
 Exporting templates
 ^^^^^^^^^^^^^^^^^^^
-As you can only save a template on to the computer you are working on you will
+As you can only save a template on to the computer you are working (*Save Locally*) on you will
```
Comment (remocrevo) [inline docs/admin/receipt_template_editor.adoc]: Looks like "(*Save Locally*)" split the phrase "working on" in half.  We'll need to move "on" back next to "working".

File: docs/admin/receipt_template_editor.adoc [hunk 2]
```diff
@@ -228,15 +232,14 @@ once you import the receipts on the new machine.
...
 receipts (i.e., more than one computer on the circulation desk, or another
 computer in the workroom that you use to checkin items or capture holds with)
 
-. Export.  
-. Select the location to save the template to, name the template, and click
-*Save*.
-. Click OK. 
+. Export Customized Templates.
+. Saves a “print_templates.json” file in the downloads folder.
+NOTE: You can rename a the file to better track your changes.
```
Comment (remocrevo) [inline docs/admin/receipt_template_editor.adoc]: Need to remove extra word "a" from "You can rename a the file".

File: docs/admin/Receipt_template_editor.adoc [hunk 3]
```diff
@@ -0,0 +1,268 @@
+Print (Receipt) Templates
```
Comment (remocrevo) [inline docs/admin/Receipt_template_editor.adoc]: This whole file seems to be an accidental duplicate of receipt_template_editor.adoc (which has changes included in this pull request). Can this file be deleted (Receipt_template_editor.adoc with capital "R")?

Comment (alynn26) [reply docs/admin/Receipt_template_editor.adoc]: Yes, it can.

File: docs/admin/receipt_template_editor.adoc (hunk 1 above)
Comment (alynn26) [reply docs/admin/receipt_template_editor.adoc]: Corrected this.

File: docs/admin/receipt_template_editor.adoc (hunk 2 above)
Comment (alynn26) [reply docs/admin/receipt_template_editor.adoc]: Corrected ths.

**Comment by remocrevo** [comment]
Thanks @alynn26 !

**Comment by alynn26** [comment]
Made some more corrections and spellings in these files, also added the information on Preferred Patron name on Receipts.

**Comment by remocrevo** [comment]
@alynn26 It looks like your glossary work is part of this pull request.  Was that intentional?

**Comment by alynn26** [comment]
No, but that works for me. Thanks.

**Comment by sandbergja** [comment]
I was looking through the glossary here -- it looks great, by the way.  I was wondering if each of the entries should have its own ID.  That way, when terms are used in the rest of the docs, we could link those terms to their definitions in the glossary.  Here's an example: http://discuss.asciidoctor.org/How-to-link-the-glossary-tp4784p4785.html

What do you think?

It also seems like the glossary file isn't included in any of the root files.

Also, it looks like this branch includes some changes to the print templates themselves.  I suspect those would be easier to review in their own commit.

**Comment by sandbergja** [comment]
I'm also getting the error `references an unknown ID "recipt_contents_reference.adoc"` when I try to compile this with a2x

**Comment by alynn26** [comment]
Seperating all pull request from this into different files.

**Pull request closed without merging**